components/Episode/EpisodeFunctions.js

```diff
@@ -0,0 +1,25 @@
+export function getCoverURL(links) {
+  if (links) {
+    const { cover_kosmos, cover_itunes, cover } = links;
+    if (cover_kosmos) {
+      return cover_kosmos;
+    } else if (cover_itunes) {
+      return cover_itunes;
+    } else if (cover) {
+      return cover;
+    } else {
+      return false;
+    }
+  }
+}
+
+export function getFormattedDate(date) {
+  if (date) {
+    const splitDate = date.split("-");
```

Comment:
You can shorten this using `date.split("-").join(".")`

Reply:
Since I also want to change the order of the date this wouldn't be working for my case.